maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+            role = None
+            content = None
+
+            if isinstance(msg, dict):
+                role = msg.get("role", "user")
+                content = msg.get("content", "")
+            else:
+                role = getattr(msg, "role", "user")
+                content = getattr(msg, "content", "")
+
+            parsed_messages.append(
+                GenerationRequestMessage(role=str(role), content=str(content))
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)